server/src/users/use-cases/registerUser/register-user.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Roles } from '@shared/constants';
+import { InMemoryUsersRepository } from '@users/test/in-memory/inMemoryUserRepository';
+import { RegisterUser } from './register-user';
+
+describe('Register collaborator', () => {
```

Comment:
Descrição está errada

"Register User", além disso deve ser possível registrar os 3 tipos de Roles cada um em um "it"